x11-libs/libxt/libxt-1.1.5.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="X Toolkit Intrinsics library"
+DESCRIPTION="libXt provides the X Toolkit Intrinsics, an abstract widget \
+library upon which other toolkits are based. Xt is the basis for many \
+toolkits, including the Athena widgets (Xaw), and LessTif (a Motif \
+implementation)."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+COPYRIGHT="1993, 1994  X Consortium
+	1989, 1998 The Open Group
+	1987, 1988 by Digital Equipment Corporation
+	1993, 2011, Oracle and/or its affiliates
+	2001,2003 Keith Packard"
+LICENSE="MIT (no promotion)"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXt-$portVersion.tar.gz"
+CHECKSUM_SHA256="b59bee38a9935565fa49dc1bfe84cb30173e2e07e1dcdf801430d4b54eb0caa3"
+SOURCE_FILENAME="libXt-portVersion.tar.gz"
+SOURCE_DIR="libXt-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=6.0.0
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libxt$secondaryArchSuffix = $portVersion
+	lib:libXt$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libICE$secondaryArchSuffix
+	lib:libSM$secondaryArchSuffix
+	lib:libX11$secondaryArchSuffix
+	# needed for TEST()
+	lib:libglib_2.0$secondaryArchSuffix
```

Comment:
Why does something needed for `TEST()` ends up here?

Reply:
Maybe someone will chock on it and write a TEST_requires? Will move it up if you prefer :)

Reply:
I mean stuff in `REQUIRES` are not activated during package `TEST()`, only `BUILD_{PRE,}REQUIRES`. If the package doesn't install anything that requires glib2, you can drop this.

Reply:
Ah, my bad, will remove